### ISSUE TITLE
ofBook Translation: Korean

### ISSUE THREAD

**Issue by bakercp** [openframeworks/ofBook#184]
Hi all,

This issue is a place to discuss and organize a Korean translation.  If you are interested or are already working on a Korean translation, please respond in below.

ping @hyok8707 @arturoc

**Comment by laserpilot** [comment]
Not Korean - but just adding a note here that someone has done a translation of the installationup4evr chapter into Japanese: https://github.com/hideyukisaito/Installation_Up_4evr - if that is another language to be tackled eventually

**Comment by arturoc** [comment]
yes i was planning to have an issue for localization in general , the issue is mostly to track the technical issues and sync with everyone that is going to work both in the korean and japanese versions.

**Comment by tpltnt** [comment]
The following chapters are pretty stable and a good start for the translation efforts:
- of_philosophy
- cplusplus_basics
- setup_and_project_structure
- intro_to_graphics
- OOPs!
- animation

**Comment by icq4ever** [comment]
I've been translating some of articles in old tutorial section in old ofSite repo.
some of ofBook's sections are finished if they're not new.
